src/actions/BitPayActions.js

```diff
@@ -147,23 +147,19 @@ export async function launchBitPay(
   }
   const invoiceInstruction = invoiceResponse.instructions[0]
   errorData = { ...errorData, invoiceInstruction }
-  if (invoiceInstruction.outputs) {
-    if (invoiceInstruction.outputs.length > 1) {
-      throw new BitPayError('MultiOutputInvoice', { errorData })
-    }
-  } else throw new BitPayError('EmptyOutputInvoice', { errorData })
-  const instructionOutput = invoiceInstruction.outputs[0]
-
+  if (!invoiceInstruction.outputs) {
```

Comment:
Need to check `outputs.length > 1`

Reply:
Do you mean > 0?

Reply:
Ah yes

Reply:
Okay it now checks for an empty array of outputs.